hack/deploy_cincinnati.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# The script serves as an example of how Cincinnati can be run on an OpenShift cluster. 
+
+# Create a new namespace/project for Cincinnati
+oc new-project cincinnati
+
+# Create a dummy secret as a workaround to not having real secrets
+oc create secret generic cincinnati-credentials --from-literal=""
+
+# Apply oc template
+oc new-app -f dist/openshift/cincinnati.yaml \
+  -p IMAGE_TAG="90efacd" \
+  -p GB_PAUSE_SECS=300 \
+  -p GB_PLUGIN_SETTINGS="$(cat <<-EOF
```

Comment:
Now that we're suggesting external folks do this sort of thing, do we need to get more serious about versioning this config file?  So Cincinnati will translate, or at least die gracefully, when it gets fed a config file with a different version than the Cincinnati binary.

Reply:
Sure, lets discuss this in a Jira card.